src/test/dynamo/Beyonce.test.ts

```diff
@@ -159,8 +159,11 @@ async function testBatchGet(jayZ?: JayZ) {
     ],
   })
 
-  sortById(results)
-  expect(results).toEqual([musician, song1, song2])
+  sortById(results.song)
+  expect(results).toEqual({
```

Comment:
This will be much nicer to work with. Thanks Josh!

Reply:
Yea definitely nicer!

If you're interested (no pressure), you can see the resulting downstream changes in our vault project: https://github.com/ginger-io/ginger-vault-service/pull/44